projects/Mallard/src/App.tsx

```diff
@@ -87,13 +87,13 @@ const shouldHavePushFailsafe = async (client: ApolloClient<object>) => {
 };
 
 const forceUpdateApiUrlIfBeta = async () => {
-	// TODO - Remove this whole function once new stack is fully in operation.
+	// TODO - Remove this whole function before release to prod.
 	if (isInBeta()) {
-		const betaApiHasSetAlready = await newApiUrlSetForBetaUsers.get();
-		if (!betaApiHasSetAlready) {
+		const rollbackedApi = await rollbackedApiUrlForBetaUsers.get();
```

Comment:
I wonder if this name is a little bit obscure. Could we use `defaultProdAPI` or something else to explain quite what this is?

Reply:
Thanks @abeddow91 - my intention was to show that the reason for this change is to roll back a previous change. Also, we don't need to this for a long time, just for enough time period so beta users who previously used temp URL from the new stack now use the proper url.

I will add a line of comments to make it a bit clearer.